torch/_inductor/graph.py

```diff
@@ -671,6 +672,23 @@ def call_function(self, target, args, kwargs):
             # passthrough lowerings from .pattern_matcher
             return target(*args, **kwargs)
 
+        def get_custom_op_layout_constraints(target, args, kwargs):
```

Comment:
All custom ops that are tagged by `lib.define` use the fallback handler to evaluate its results. If a custom op happens to have an inductor lowering instead, the original op never gets called, and it will also control its own layout constraints directly through the IR.